app/src/marketplace/containers/ProductPage/CommunityStats.jsx

```diff
@@ -0,0 +1,251 @@
+import Dropdown from '$shared/components/Dropdown'
+import MembersGraph from '$mp/components/ProductPage/MembersGraph'
+
+import styles from './communityStats.pcss'
+
+const initialStats = {
+    revenue: {
+        label: 'Total product revenue',
+        unit: 'DATA',
+        loading: true,
+    },
+    members: {
+        label: 'Active Members',
+        loading: true,
+    },
+    averageRevenue: {
+        label: 'Avg rev member / month',
+        unit: 'DATA',
+        loading: true,
+    },
+    subscribers: {
+        label: 'Subscribers',
+        loading: true,
+    },
+    adminFee: {
+        label: 'Admin Fee',
+        unit: '%',
+        loading: true,
+    },
+    created: {
+        label: 'Product created',
+        loading: true,
+    },
+}
+
+const CP_SERVER_POLL_INTERVAL_MS = 10000
+const MILLISECONDS_IN_MONTH = 1000 * 60 * 60 * 24 * 30
+
+const CommunityStats = () => {
+    const product = useProduct()
+    const contractProduct = useContractProduct()
+    const [stats, setStats] = useState(initialStats)
+    const [totalEarnings, setTotalEarnings] = useState(null)
+    const [memberCount, setMemberCount] = useState(null)
+    const [shownDays, setShownDays] = useState(7)
+    const isMounted = useIsMounted()
+
+    const { communityDeployed, created, beneficiaryAddress } = product
+    const community = useCommunityProduct()
+
+    const { subscriberCount } = contractProduct || {}
+    const { adminFee, joinPartStreamId } = community || {}
+
+    useEffect(() => {
+        if (totalEarnings) {
+            setStats((prev) => ({
+                ...prev,
+                revenue: {
+                    ...prev.revenue,
+                    value: fromAtto(totalEarnings).toNumber() || 0,
+                    loading: false,
+                },
+            }))
+        }
+    }, [totalEarnings])
+
+    useEffect(() => {
+        setStats((prev) => ({
+            ...prev,
+            subscribers: {
+                ...prev.subscribers,
+                value: subscriberCount || 0,
+                loading: false,
+            },
+        }))
+    }, [subscriberCount])
+
+    useEffect(() => {
+        if (adminFee) {
+            setStats((prev) => ({
+                ...prev,
+                adminFee: {
+                    ...prev.adminFee,
+                    value: (adminFee * 100).toFixed(0),
+                    loading: false,
+                },
+            }))
+        }
+    }, [adminFee])
+
+    useEffect(() => {
+        if (created) {
+            setStats((prev) => ({
+                ...prev,
+                created: {
+                    ...prev.created,
+                    value: new Date(created).toLocaleDateString(),
+                    loading: false,
+                },
+            }))
+        }
+    }, [created])
+
+    useEffect(() => {
+        if (totalEarnings && created && memberCount) {
+            const productAgeMs = Date.now() - new Date(created || 0).getTime()
```

Comment:
`created || 0` can never happen due to `if (totalEarnings && created && memberCount)`

Reply:
Fixed in https://github.com/streamr-dev/streamr-platform/pull/755/commits/3c987b57e16f21a9c0613541f27b42850c4376ce